modules/core/output_modules.php

```diff
@@ -177,28 +183,60 @@ class Hm_Output_login_start extends Hm_Output_Module {
      * Looks at the current login state and outputs the correct form
      */
     protected function output() {
+        $config = new Hm_Site_Config_File(CONFIG_FILE);
+        $fancy_login = $config->get('fancy_login','n');
+        if($fancy_login=='n'):
+            if (!$this->get('router_login_state')) {
+                $css = '<style type="text/css">.mobile .login_form{margin-top:60px;display:block;float:none;width:100%;'.
+                    'background-color:#fff;font-size:130%;height:auto;}.logged_out{display:block !important;}.sys_messages'.
+                    '{position:fixed;right:20px;top:15px;min-height:30px;display:none;background-color:#fff;color:teal;'.
+                    'margin-top:0px;padding:15px;padding-bottom:5px;white-space:nowrap;border:solid 1px #999;border-radius:'.
+                    '5px;filter:drop-shadow(4px 4px 4px #ccc);z-index:101;}.g-recaptcha{margin-left:-12px;}.mobile .g-recaptcha{'.
+                    'clear:left;margin-left:20px;}.title{font-weight:normal;padding:0px;margin:0px;margin-left:20px;'.
+                    'margin-bottom:20px;letter-spacing:-1px;color:#999;}html,body{max-width:100%;min-height:100%;'.
+                    'background-color:#fff;}body{background:linear-gradient(180deg,#faf6f5,#faf6f5,#faf6f5,#faf6f5,'.
+                    '#fff);font-size:1em;height:100%;color:#333;font-family:Arial;padding:0px;margin:0px;min-width:700px;'.
+                    'font-size:100%;}input,option,select{font-size:100%;padding:3px;}textarea,select,input{border:solid '.
+                    '1px #ddd;background-color:#fff;color:#333;border-radius:3px;}.screen_reader{position:absolute'.
+                    ';top:auto;width:1px;height:1px;overflow:hidden;}.login_form{float:left;font-size:90%;'.
+                    'padding-top:60px;height:300px;border-radius:0px 0px 20px 0px;margin:0px;background-color:#f5f5f5;'.
+                    'width:300px;padding-left:20px;}.login_form input{clear:both;float:left;padding:4px;margin-left:20px;'.
+                    'margin-top:10px;margin-bottom:10px;}#username,#password{width:200px;}.err{color:red !important;}.long_session'.
+                    '{float:left;}.long_session input{padding:0px;float:none;}.mobile .long_session{float:left;clear:both;}</style>';
+    
+            return $css.'<form class="login_form" method="POST">';
```

Comment:
Indentation

Reply:
What I meant by indentation was that you had return statement indented the wrong level compared to the nested if statement. Now the diff changes show automated syntax updates in the whole file which is not desired. Please only update code that you change and leave the whitespace or bracket refactoring for its own task if and when we decide to do so. I see you placed opening brackets on a new line, added spaces around concatenation operators and such things - we don't want to disturb the code diff with so much stuff. If we decide to support PSR-4 or another standard, we will do in a separate PR and do for the whole codebase. For now, please just update the places in code that you do functional updates and try to use the syntax formatting in line with the whole project.

Reply:
Thanks @kroky , i now understand what you meant